docs/magicTagTypes.md

```diff
@@ -51,6 +51,35 @@ Inserts the unescaped value of `expression` into the result.
 <div><b>Justin</b></div>
 ```
 
+This also works for rendering [can-component] instances:
+
+```js
+import Component from "can-component";
+import stache from "can-stache";
+
+const MyGreeting = Component.extend({
+  tag: "my-greeting",
+  view: "<p>Hello {{subject}}</p>",
+  ViewModel: {
+    subject: "string"
+  }
+});
+
+const myGreetingInstance = new MyGreeting({
+  viewModel: {
+    subject: "friend"
+  }
+});
+
+const template = stache("<div>{{{componentInstance}}}</div>");
+
+const fragment = template({
+  componentInstance: myGreetingInstance
+});
+
+fragment; //-> <div><p>Hello friend</p></div>
```

Comment:
When you do this, there is no `<my-greeting>` tag?

Reply:
There is, thanks for pointing out my mistake! It’s fixed now.